### ISSUE TITLE
Hand Camera Calibration

### ISSUE THREAD

**Issue by pet1330** [LCAS/zoidbot#4]
The hand cameras need to be calibrated for the circle detection to work correctly.

**Comment by pet1330** [comment]
**TL;DR** It was WhyCon, not the camera...

I went back and had a look at this issue, it turns out we were using the K camera matrix rather than the P matrix, which does not take into account the position offset of the image. This did not affect the external cameras as when their resolution was changed, the K matrix is updated accordingly, but changing the resolution in Baxter does not actually change the resolution of the image that is taken, but instead just crops out the centre of the image to reduce its size. As a result, the K matrix is only representative on the full image and the P matrix is required to get the cropped offset that is returned.
